### PR TITLE
docs(example): Waiter code walkthrough on Codeflow

### PR DESCRIPTION
**Pull request by mokshjawa**
A quick PR to add a code walkthrough to Waiter using [Codeflow](https://usecodeflow.com/).

Check out the walkthrough here: https://usecodeflow.com/tutorials/view/twosigma/waiter/tree/cb1000/ck8diy2g500000ul4s0b6l4zb

### CODE REVIEW

File: waiter/README.md [hunk 1]
```diff
@@ -58,6 +58,10 @@ $ lein test :integration-slow
 $ WAITER_TEST_REQUEST_LATENCY_MAX_INSTANCES=30 APACHE_BENCH_DIR=/usr/bin lein test :perf
 ```
 
+## Code Walkthrough
+
+For more information on how Waiter is set up, check out this [code walkthrough](https://usecodeflow.com/tutorials/view/twosigma/waiter/tree/cb1000/ck8diy2g500000ul4s0b6l4zb).
```
Comment (shamsimam) [inline waiter/README.md:63]: Walkthrough looks neat.

Does this link point to an immutable resource? That is are we sure the content isn't going to change?

Comment (shamsimam) [reply waiter/README.md:63]: Also, can we monospace fonts for code snippets?

Comment (mokshjawa) [reply waiter/README.md:63]: Yup, immutable. I'll make you the owner of the walkthrough so only you can change it (when you login at https://usecodeflow.com/). Updated the fonts!

**Comment by mokshjawa** [comment]
@shamsimam any other changes you'd like to see?

**Comment by shamsimam** [comment]
I'm contemplating whether using a beakerx notebook is better for this as it also allows me to run code live when required. It can also be version controlled in the same repository.

I'm not sure I want to merge this though, that adds additional burden of having to maintain the document in codeflow along with the other places we maintain documentation.

**Comment by DaoWen** [comment]
This was a cool idea, but as Shams noted above, we don't plan to maintain this. Closing.

**Pull request closed without merging**